apps/portal-front/src/components/service/document/one-click-deploy/useOneClickDeployTab.tsx

```diff
@@ -9,13 +10,13 @@ import { toast } from 'filigran-ui';
 import { useTranslations } from 'next-intl';
 import { useMutation } from 'react-relay';
 
-const URL_CONFIGS = {
+const URL_CONFIGS_OPENCTI = {
```

Comment:
Nitpicking but I would have called it:
```suggestion
const OPENCTI_URL_CONFIGS = {
```